include/aspect/simulator_signals.h

```diff
@@ -52,8 +52,7 @@ namespace aspect
    * not actually just pointers to functions, but std::function objects
    * that have a certain signature. Consequently, they can have much more
    * complicated types than just function pointers, such as objects with
-   * an <code>operator()</code> or function calls treated with things
-   * like std::bind.
+   * an <code>operator()</code> or lambdas.
```

Comment:
lambdas -> lambda functions. Let's at least spell it out so that people can google for it.